src/types/KeycloakUserId.ts

```diff
@@ -0,0 +1,41 @@
+import { Newtype, iso as createNewtypeIsomorphism } from 'newtype-ts';
```

Comment:
Haha, now that I know what it stands for I understand why they went with `iso`!

Reply:
The irony lol (I can keep it as iso if you prefer, but it felt totally opaque to me 😅)